src/library/scala/collection/immutable/Vector.scala

```diff
@@ -1156,6 +1168,14 @@ private[immutable] trait VectorPointer[+T] {
       }
     }
 
+    private final def nullSlotAndCopy0(array: Array[Array[AnyRef]], index: Int, destination: Array[AnyRef] = null): Array[AnyRef] = {
```

Comment:
TODO: Include Scaladocs to explain that this is a specialized form of `nullSlotAndCopy` specialized for `display0` which exists solely for avoiding use of `array.clone()` which may be a shared array with element type != `java.lang.Object`